src/catleg/query.py

```diff
@@ -110,6 +110,52 @@ def get_backend(spec: str):
     return LegifranceBackend(client_id, client_secret)
 
 
+class LegifranceArticle(Article):
+    def __init__(self, id: str, text: str, text_html: str, nota: str, nota_html: str):
+        self._id = id
+        self._text = text
+        self._text_html = text_html
+        self._nota = nota
+        self._nota_html = nota_html
+
+    @property
+    def id(self) -> str:
+        return self._id
+
+    @property
+    def text(self) -> str:
+        return self._text
+
+    @property
+    def text_html(self) -> str:
+        return self._text_html
+
+    @property
+    def nota(self) -> str:
+        return self._nota
+
+    @property
+    def nota_html(self) -> str:
+        return self._nota_html
+
+    @property
+    def type(self) -> ArticleType:
+        return parse_article_id(self.id)[0]
+
+    def to_markdown(self) -> str:
+        text_md = md(_clean_legifrance_html(self.text_html))
+        if len(self.nota_html):
+            nota_md = md(_clean_legifrance_html(self.nota_html))
+            text_md += f"NOTA :\n\n{nota_md}"
```

Comment:
Shouldn't it be `\n\nNOTA :\n\n` so that the NOTA is also newline-separated from the text before?

Reply:
Yes, usually there seems to be an abundance of newlines, empty paragraphs etc. in the legifrance html, but for clarity and consistency it would make sense to strip spaces at the beginning and the end of `text_html` and to add explicit newlines before the nota ; I'll do this before the merge. Thanks!